src/crate/client/sqlalchemy/tests/dict_test.py

```diff
@@ -41,7 +40,8 @@ class SqlAlchemyDictTypeTest(TestCase):
 
     def setUp(self):
         self.engine = sa.create_engine('crate://')
-        metadata = sa.MetaData()
+        # FIXME: Deprecated with SA20.
+        metadata = sa.MetaData(bind=self.engine)
```

Comment:
This is the only spot where the test suite still croaks when removing the `bind` argument. Still, I would like to bring in this patch, in order to have a common ground for further activities, maybe together with @hammerhead.